packages/edit-site/src/utils/find-template.js

```diff
@@ -18,7 +18,9 @@ const { fetch } = window;
  */
 export default async function findTemplate( path, getEntityRecords ) {
 	const { data } = await fetch(
```

Comment:
Shouldn't this use apiFetch instead? I believe it might solve the issue as well?

Reply:
We'd have to move the endpoints to use the REST API instead.

Reply:
I'm not sure this is a requirement for apiFetch?

Reply:
I see no option in apiFetch to change the root URL. In this case, the root URL would be the site URL instead of the REST API URL.